### DISCUSSION TITLE
Backing Up s3 files

### DISCUSSION THREAD

**Discussion by makoy71692** [spatie/laravel-backup#1647] [General]
Hello is there a way to include files like images file that are saved on S3 system to the creation of back up?

like in our case, we also use your spatie-medialibrary and have those media saved in our Minio filesystem. we would like to include those files on the creation of our backup

**Comment by parallels999** [comment]
Yes, you can add everything you need in the settings

**Reply by makoy71692** [reply]
hello is there a guide for including S3 files in the backup
